HW_4/task_4.py

```diff
@@ -0,0 +1,125 @@
+import os
+from pathlib import Path
+import requests
+import time
+import argparse
+import threading
+from multiprocessing import Process
+import asyncio
+import aiohttp
+
+images = []
+with open('images.txt', 'r') as f:
+    for image in f.readlines():
```

Comment:
Файловый дескриптор уже является итератором.
```suggestion
    for image in f:
```